Status V2 Re-mapping

TheKentuckian opened the pull request.

TheKentuckian committed:
Status V2 Re-mapping

marcinbujar commented:
Hey, I thought it might be a good idea to refactor this slightly and only store the status definition in one place, (e.g. a constants.ts file in the top-level module) so that other code which needs it does not need to redefine it in several places.

TheKentuckian committed:
Add 'Device in for repair' status

TheKentuckian committed:
Configurize the auth audience value

TheKentuckian committed:
Temporarily set API to api.testing...

This is a bit hacky, and intended to be temporary until I can figure out a clean way to parameterize the auth audience in the auth service

TheKentuckian committed:
Reverting production audience change

TheKentuckian committed:
Copying device back for repair to other modules

TheKentuckian committed:
Fixing new status styling, and shifting colours slightly

TheKentuckian committed:
Factor out kit status labels

TheKentuckian commented:
> Hey, I thought it might be a good idea to refactor this slightly and only store the status definition in one place, (e.g. a constants.ts file in the top-level module) so that other code which needs it does not need to redefine it in several places.

Good shout. I've factored out the label/value pairs in the same way we were already sharing KIT_STATUS

TheKentuckian committed:
Merge branch 'master' into status_v2
